cmake/Geant3RequiredPackages.cmake

```diff
@@ -14,21 +14,23 @@
 
 #-- ROOT (required) ------------------------------------------------------------
 find_package(ROOT CONFIG REQUIRED)
+set(ROOT_DEPS ROOT::Core ROOT::RIO ROOT::Net ROOT::Hist ROOT::Graf ROOT::Graf3d ROOT::Gpad
+    ROOT::Tree ROOT::Rint ROOT::Postscript ROOT::Matrix ROOT::Physics ROOT::MathCore
+    ROOT::Thread ROOT::Geom ROOT::EG)
```

Comment:
This list may be probably reduced; the candidates for removal are ROOT::Net ROOT::Hist ROOT::Graf ROOT::Graf3d ROOT::Gpad ROOT::Postscript ROOT::Matrix.